src/main/java/com/iexec/blockchain/chain/ChainConfig.java

```diff
@@ -16,60 +16,52 @@
 
 package com.iexec.blockchain.chain;
 
-import com.iexec.common.chain.validation.ValidNonZeroEthereumAddress;
-import jakarta.annotation.PostConstruct;
-import jakarta.validation.*;
-import jakarta.validation.constraints.Max;
-import jakarta.validation.constraints.NotEmpty;
-import jakarta.validation.constraints.NotNull;
-import jakarta.validation.constraints.Positive;
+import com.iexec.commons.poco.chain.validation.ValidNonZeroEthereumAddress;
+import jakarta.validation.constraints.*;
 import lombok.Builder;
 import lombok.Value;
 import lombok.extern.slf4j.Slf4j;
 import org.hibernate.validator.constraints.URL;
+import org.hibernate.validator.constraints.time.DurationMax;
+import org.hibernate.validator.constraints.time.DurationMin;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
 
-import java.util.Set;
+import java.time.Duration;
 
 @Slf4j
 @Value
 @Builder
+@Validated
 @ConfigurationProperties(prefix = "chain")
 public class ChainConfig {
 
     @Positive(message = "Chain id should be positive")
     @NotNull
     int id;
 
+    boolean sidechain;
+
     @URL
     @NotEmpty
     String nodeAddress;
 
-    @Positive(message = "Block time should be positive")
-    @NotNull
-    int blockTime;
-
     @ValidNonZeroEthereumAddress
     String hubAddress;
 
-    boolean isSidechain;
+    @DurationMin(millis=100)
+    @DurationMax(seconds=20)
```

Comment:
Auto formatting display them as follow on my side
```suggestion
    @DurationMin(millis = 100)
    @DurationMax(seconds = 20)
```

Reply:
Resolved in [2c4e9ca](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/171/commits/2c4e9ca4d1ec0553fa571dd70d580e988e7c4654)